04-python/solutions/generate_password_constrained2.py

```diff
@@ -11,9 +11,9 @@
 def generate_password_constrained2():
     size = randint(10, 20)
     n_digits = randint(1, len(digits))
```

Comment:
Cool, then we could also probably update this to `randint(1, size)`? With repetitions, we can have more than `len(digits)`.

Reply:
:thumbsup: (or `randint(1, size-2)`)